site/source/_posts/2020-01-13-announcing-embark-5.md

```diff
@@ -0,0 +1,165 @@
+
+Here's what such a config looked like prior to v5:
+
+```js
+...
+deployment: {
+  host: "localhost", // Host of the blockchain node
+  port: 8546, // Port of the blockchain node
+  type: "ws" // Type of connection (ws or rpc),
+  accounts: [...]
+},
+...
+```
+
+There's no equivalent for this configuration inside the Smart Contract configuration file anymore, so this section can just be entirely (re)moved.
+
+### `contracts` property has been renamed to `deploy`
+
+When configuring Smart Contracts, there are a few deployment hooks that can be specified, such as `beforeDeploy` and `afterDeploy`. To make the API a bit more descriptive and clarifying the intentions, the `contracts` property has been renamed to `deploy`, aligning wonderfully with the deployment hooks counterparts.
+
+Before:
+
+```js
+...
+contracts: {
+  SimpleStorage: {
+    fromIndex: 0,
+    args: [100],
+    onDeploy: async () => { ... },
+    deployIf: async () => { ... }
+  }
+}
+...
+```
+
+After:
+
+```js
+...
+deploy: {
+  SimpleStorage: {
+    fromIndex: 0,
+    args: [100],
+    onDeploy: async () => { ... },
+    deployIf: async () => { ... }
+  }
+}
+...
+```
+
+## Polished blockchain configuration API
+
+One of the most conplex APIs has been Embark's blockchain configuration API. That's why we've put a lot of effort into streamlining the settings and properties and removing the ones that happened to be redundant. On top of that, Embark not defines most of them as defaults, resulting in significantly smaller and less complex configuration files.
+
+The following configuration properties have been renamed:
+
+- `isDev` is now `miningMode: 'dev'`
+- `mineWhenNeeded` is now `miningMode: 'auto'`
+- `ethereumClientName` is now `client`
+
+We've also remove several endpoint related settings such as `host`, `port` etc. and replaced them with a single `endpoint` property. Here's what the new defaults look like:
+
+```js
+module.exports = {
+  default: {
+    enabled: true,
+    client: "geth"
+  },
+  development: {
+    clientConfig: {
+      miningMode: 'dev'
+    }
+  },
+  testnet: {
+    endpoint: "https://external-node.com",
+    accounts: [
+      {
+        mnemonic: "12 word mnemonic"
+      }
+    ]
+  }
+}
+```
+
+For a more information on the blockchain configuration, head over to the [official docs](/docs/blockchain_configuration.html).
+
+## Accounts configuration moved to blockchain config
+
+Prior to Embark v5 it was possible to specify and configure various different accounts for deployment and interaction inside the Smart Contract configuration, as well as the blockchain configuration. This caused a lot of confusion as it wasn't really clear which accounts belonged to what action. To aid against this confusion, we've moved the accounts configuration entirely to the blockchain configuration as well, making it much more straight forward to find the right place when setting up custom accounts.
```

Comment:
Above, "Embark 5" is used consistently but here it's `Embark v5` — I think either way is fined but it should be consistent throughout the blog post.

```suggestion
Prior to Embark 5 it was possible to specify and configure various accounts for deployment and interaction both inside the Smart Contract configuration and the blockchain configuration. This caused a lot of confusion because it wasn't really clear which accounts belonged to what action. To eliminate confusion, we've moved the accounts configuration entirely to the blockchain configuration, making it much more straightforward to find the right place when setting up custom accounts.
```